types.go

```diff
@@ -113,6 +113,17 @@ type xmlInfo struct {
 	XmlSchemaCollection string
 }
 
+// Kerberos Client State
+type krb5ClientState int
+
+const (
+	/* initiator states */
```

Comment:
Single line comment, initial capital in comment.
Unless the consts are needed by the client, these should be private (lowercase).